src/test/resources/config/config_loader_test/default.yml

```diff
@@ -0,0 +1,7 @@
+"/default":
+  ".conf.yml":
+    refs:
+      users:
+        avatar_id: images
+      "content.comment":
+        ticket_id: tickets
```

Comment:
Add a new line

Reply:
I added one but github does not display that 😢

Reply:
no worries